ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/util/Time.java

```diff
@@ -8,6 +8,9 @@
 
 public class Time {
 
+  private Time() {
+  }
```

Comment:
a private empty constructor indicates to both users and jacoco that this class in not meant to be instantiated.